packages/test/amplify_integration_test/lib/src/stubs/amplify_stub.dart

```diff
@@ -1,6 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+// ignore_for_file: invalid_use_of_internal_member
```

Comment:
*nit:* Not a bug deal since this is a test, but I would add this for the one line it is needed rather than the full file. That way we don't accidentally start using internal APIs without intending to.

Reply:
Updated